.github/workflows/copilot_deploy.yml

```diff
@@ -389,7 +389,7 @@ jobs:
       run: |
         yq -i '.image.location = "ghcr.io/communitiesuk/funding-service-design-notification:${{ github.ref_name == 'main' && 'latest' || needs.tag_version.outputs.version_to_tag }}"'  copilot/fsd-notification/manifest.yml
 
-    - name: Copilot production deploy
+    - name: Copilot prod deploy
```

Comment:
This is just the name of the job - on the one hand, it IS production, on the other hand, keep it consistent, as it IS deploying the _copilot env_ called prod